doc/developers.md

```diff
@@ -31,10 +31,45 @@ Cassandra and Spark nodes and are the core of our test coverage.
 
 ### Merge Path
 
-b2.5 => Master
+b2.5 => b3.0 => master
 
-New features can be considered for 2.5 as long as they do not break apis
-In general 3.0 should be the target for new features
+New features can be considered for 2.5 as long as they do not break apis.
+Once a feature is ready for b2.5, create a feature branch for b3.0 and merge
+b2.5 feature branch to b3.0 feature branch. Repeat for master. 
+
+Example for imaginary SPARKC-9999.
+
+Let's assume that `datastax` is git@github.com:datastax/spark-cassandra-connector.git remote 
+and origin is your personal clone.
+```shell
+$ git remote -v
+datastax	git@github.com:datastax/spark-cassandra-connector.git (fetch)
+datastax	git@github.com:datastax/spark-cassandra-connector.git (push)
+...
+```
+
+Here is how the work should look like.
+
+```shell
+git fetch datastax
```

Comment:
The steps below assumes the person has push access to the github.com/spark-cassandra-connector, but usually we will request folks to _fork_ the repo, make their changes, push their branch and create a PR IIUC. So, should we add a separate section for the instructions for those that fork and create PRs? Just a thought.

Reply:
origin is individual clone, datastax is the main repo, it's mentioned somewhere above